README.md

```diff
@@ -27,6 +30,39 @@ Import stylesheets and require javascripts:
 //= require active_admin/searchable_select
 ```
 
+##### Using assets via Webpacker (or any other assets bundler) as a NPM module (Yarn package)
+
+Execute:
+
+    $ npm i @codevise/activeadmin-searchable_select
+
+Or
+
+    $ yarn add @codevise/activeadmin-searchable_select
+
+Or add manually to `package.json`:
+
+```json
+"dependencies": {
+  "@codevise/activeadmin-searchable_select": "1.6.0"
+}
+```
+and execute:
+
+    $ yarn
+
+Add the following line into `app/assets/javascripts/active_admin.js`:
+
+```javascript
+import '@codevise/activeadmin-searchable_select';
+```
+
+Add the following line into `app/assets/stylesheets/active_admin.scss`:
+
+```css
+@import '@codevise/activeadmin-searchable_select';
+```
+
```

Comment:
Which asset pipeline does this represent? Wouldn't Webpacker code live under `app/javascript`?

Reply:
Webpacker code can live in any places, this only depended on project configuration or settings in `webpacker.yml` etc..  In the current example assets are provided via Webpacker. If you mean should we provide full or relative path to npm package placed in `node_modules` - this depends from configuration, by default Webpacker try to find imported assets in `node_modules`, `app/assets` folders without any additional configuration/manipulation. But of course, you can import assets with a full or relative path (example: `@import './path-to-node_modeuls/@codevise/activeadmin-searchable_select';`) or add target folder to search assets to you Webpacker configuration (`webpacker.yml`):
```
default: &default
  # Additional paths webpack should lookup modules
  # ['app/assets', 'engine/foo/app/assets']
  additional_paths: [engine/foo/app/assets]
```

Reply:
What I meant was: In its default configuration, Webpacker (and - far as I'm aware - also the other solutions like jsbundling-rails and importmap-rails) places its files in `app/javascript`. `app/assets` made me think of Sprockets and I was wondering if there is a way now to also use imports in Sprockets directly. If there isn't, it might be less confusing to also use `app/javascript` in the example.

Reply:
Understood. `imports in Sprockets directly` not supported (and no easy way to do it). Yes, it makes sense to be more precise, replace `Add the following line into app/assets/javascripts/active_admin.js:` on something like that `Add the following line into `app/javascript/active_admin.js`:` and the same for stylesheets. 
Create PR for this one?

Reply:
That would be great! :+1: